main.js

```diff
@@ -0,0 +1,125 @@
+$(document).ready(function(){
+
+    $('#square').hide();
+
+    //$( "*", document.body ).click(function( event ) {
+    //    var offset = $( this ).offset();
+    //    event.stopPropagation();
+    //    $( "#result" ).text( this.tagName +
+    //        " coords ( " + offset.left + ", " + offset.top + " )" );
+    //});
+
+    function render(val){
+        console.log(val);
+        $('#square').css({
+           "width" : val.width,
+            "height": val.height,
+            "background" : val.background,
+            "left": val.left + "%",
+            "top": val.top + "%",
+            "textAlign": val.textAlign,
+            "border": val.border,
+            "marginLeft": val.marginLeft,
+            "marginTop": val.marginTop,
+            "position": val.position
+        });
+    }
+
+
+    $('#btn').on('click', function(){
+
+        $.ajax({
+            url: 'http://demo6147751.mockable.io/ninosla',
+            method: 'GET',
+            dataType: 'json',
+            success: function(data){
+                render(data);
```

Comment:
Validate returned data and then render. For example the API might not return any data for height, and then your render function won't work. So the point is - asign default values if the API does not return them, and notify the user somehow